uaclient/entitlements/fips.py

```diff
@@ -207,30 +207,31 @@ def prompt_if_kernel_downgrade(
 
     def install_packages(
         self,
+        progress: api.ProgressWrapper,
         package_list: Optional[List[str]] = None,
         cleanup_on_failure: bool = True,
-        verbose: bool = True,
     ) -> None:
         """Install contract recommended packages for the entitlement.
 
         :param package_list: Optional package list to use instead of
             self.packages.
         :param cleanup_on_failure: Cleanup apt files if apt install fails.
-        :param verbose: If true, print messages to stdout
         """
-
-        if verbose:
-            event.info(
-                messages.INSTALLING_SERVICE_PACKAGES.format(title=self.title)
-            )
-
         # We need to guarantee that the metapackage is installed.
         # While the other packages should still be installed, if they
         # fail, we should not block the enable operation.
         mandatory_packages = self.packages
-        super().install_packages(
-            package_list=mandatory_packages, verbose=False
-        )
+        if mandatory_packages:
```

Comment:
Is there a situation where this will be empty ?

Reply:
On containers, this will be empty.